dts/bindings/serial/st,stm32-uart-base.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 STMICROELECTRONICS
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for STM32 UART peripherals.
+description: STM32 UART-BASE
+
+compatible: "st,stm32-uart-base"
+
+include: [uart-controller.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    clocks:
+      required: true
+
+    interrupts:
+      required: true
+
+    single-wire:
```

Comment:
Does single-wire apply to LPUART also? If not, this should not be in the base yaml.

Reply:
@dcpleung LPUART and U(S)ART have the same base and majority of operations are performed the same way, Lpuart support single_wire.